### PR TITLE
set up layout and connect db

### PR DESCRIPTION
**Pull request by w286432531**
Changed folder names
Set up basic layout and router
Set up database connection

### CODE REVIEW

File: app-web/src/components/pages/Error.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { Container } from "react-bootstrap";
+const ErrorPage = () => {
+  return <Container>
+    <h1>There is a error</h1>
+  </Container>;
```
Comment (linmao0991) [inline app-web/src/components/pages/Error.js]: Let's wrap this `return` in `()`. The error should be a bit more descriptive. Looks like this route is a catch for routes that doesn't exist so maybe something like `404 Page not found.`

File: app-web/src/components/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Button, Card } from 'react-bootstrap';
+
+const Home = () => {
+    return <Card>
+        Hello This is home
+        <Button>
+            click here
```
Comment (linmao0991) [inline app-web/src/components/pages/Home.js]: Let's wrap this `return` with `()`

File: prisma/schema.prisma [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+datasource db {
+  provider = "postgresql"
+  url      = env("LOCAL_DATABASE")
+  // shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
+}
+
+model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
```
Comment (linmao0991) [inline prisma/schema.prisma:17]: Is `name` a persons name or a user name? If it is a name it should be two fields `firstName` and `lastName`

File: app-web/src/components/global/Header.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            <Nav className="ms-auto">
+              <Nav.Link as={NavLink} to="/">
+                Home
+              </Nav.Link>
+              <Nav.Link as={NavLink} to="/about-us">
+                About Us
+              </Nav.Link>
+              <Nav.Link as={NavLink} to="/menu">
+                menu
+              </Nav.Link>
```
Comment (linmao0991) [inline app-web/src/components/global/Header.js:24]: Let's capitalize `menu` to `Menu` to keep style with the other nav links.

File: app-web/src/components/global/Layout.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import Footer from "./Footer";
+import Routers from "./Routers";
+import { Fragment } from "react";
+
+const Layout = () => {
+  return (
+    <Fragment>
+      <Header />
+      <main>
+        <Routers />
```
Comment (linmao0991) [inline app-web/src/components/global/Layout.js:11]: There is a shorter syntax for `<Fragment></Fragment>` which is just `<></>`.

File: app-web/src/components/global/Layout.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { BrowserRouter as Router, Routes, Route, Link } from "react-router-dom";
+import Header from "./Header";
+import Footer from "./Footer";
+import Routers from "./Routers";
+import { Fragment } from "react";
+
```
Comment (linmao0991) [inline app-web/src/components/global/Layout.js:5]: Let's import `react` as the first import.

**Review by linmao0991 — CHANGES_REQUESTED** [review]
Usually `prisma` related things should go into the api folder. If you can get the `prisma` folder into `app-api` then that would be better, but otherwise it's not a big deal. 

You should also put the `.vscode` folder into your `.gitignore` unless this is something you want shared between developers.

File: prisma/schema.prisma (hunk 3 above)
Comment (w286432531) [reply prisma/schema.prisma:17]: This is just a dummy test to see if migration works.

**Review by linmao0991 — APPROVED** [review]
✅

**Pull request merged**